Makefile

```diff
@@ -32,10 +33,15 @@ cross: clean
 .PHONY: vendor
 vendor: vendor.conf
 	@vndr 2> /dev/null
-	@script/validate/check-git-diff vendor
+	@scripts/validate/check-git-diff vendor
+
+## Generate man pages from go source and markdown
+.PHONY: manpages
+manpages:
+	@man/generate.sh
```

Comment:
Yes, I think that's how it works. I did it for `vendor` (in this Makefile) and it seems to work so far.